clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/data/task/jedis/JedisTask.java

```diff
@@ -0,0 +1,162 @@
+package com.netflix.spinnaker.clouddriver.data.task.jedis;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.google.common.collect.Iterables;
+import com.netflix.spinnaker.clouddriver.data.task.SagaId;
+import com.netflix.spinnaker.clouddriver.data.task.Status;
+import com.netflix.spinnaker.clouddriver.data.task.Task;
+import com.netflix.spinnaker.clouddriver.data.task.TaskState;
+import java.util.List;
+import java.util.Set;
+import javax.annotation.Nonnull;
+import org.codehaus.groovy.runtime.DefaultGroovyMethods;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@JsonPropertyOrder({"status", "history"})
+public class JedisTask implements Task {
```

Comment:
there's a pretty gnarly comment at the top of the groovy version that I suspect is still applicable in the java version, worth preserving?

Reply:
Ah, yeah. Good spot - I guess it wasn't brought along for the ride because it wasn't doc block?